docs/concepts/policy/pod-security-policy.md

```diff
@@ -125,141 +387,153 @@ to the volume sources that are defined when creating a volume:
+root filesystem (i.e. no writeable layer).
 
-{% include code.html language="yaml" file="psp.yaml" ghlink="/docs/concepts/policy/psp.yaml" %}
+### Users and groups
 
-Create the policy by downloading the example file and then running this command:
+**RunAsUser** - Controls the what user ID containers run as.
 
-```shell
-$ kubectl create -f ./psp.yaml
-podsecuritypolicy "permissive" created
-```
-
-## Getting a list of Pod Security Policies
+- *MustRunAs* - Requires a `range` to be configured. Uses the first value
+of the range as the default. Validates against the configured range.
+- *MustRunAsNonRoot* - Requires that the pod be submitted with a non-zero
+`runAsUser` or have the `USER` directive defined in the image. No default
+provided. Setting `AllowPrivilegeEscalation=false` is strongly recommended with
+this strategy.
+- *RunAsAny* - No default provided. Allows any `runAsUser` to be specified.
 
-To get a list of existing policies, use `kubectl get`:
+**SupplementalGroups** - Controls which group IDs containers add.
 
-```shell
-$ kubectl get psp
-NAME        PRIV   CAPS  SELINUX   RUNASUSER         FSGROUP   SUPGROUP  READONLYROOTFS  VOLUMES
-permissive  false  []    RunAsAny  RunAsAny          RunAsAny  RunAsAny  false           [*]
-privileged  true   []    RunAsAny  RunAsAny          RunAsAny  RunAsAny  false           [*]
-restricted  false  []    RunAsAny  MustRunAsNonRoot  RunAsAny  RunAsAny  false           [emptyDir secret downwardAPI configMap persistentVolumeClaim projected]
-```
-
-## Editing a Pod Security Policy
+- *MustRunAs* - Requires at least one range to be specified. Uses the
+minimum value of the first range as the default. Validates against all ranges.
+- *RunAsAny* - No default provided. Allows any `supplementalGroups` to be
+specified.
 
-To modify policy interactively, use `kubectl edit`:
+### Privilege Escalation
+
+These options control the `AllowPrivilegeEscalation` container option. This bool
+directly controls whether the
+[`no_new_privs`](https://www.kernel.org/doc/Documentation/prctl/no_new_privs.txt)
+flag gets set on the container process. This flag will prevent `setuid` binaries
+from changing the effective user ID, and prevent files from enabling extra
+capabilities (e.g. it will prevent the use of the `ping` tool). This behavior is
+required to effectively enforce `MustRunAsNonRoot`.
+
+It defaults to `nil`. The default behavior of `nil` allows privilege escalation
+so as to not break setuid binaries. Setting it to `false` ensures that no child
+process of a container can gain more privileges than its parent.
+
+**AllowPrivilegeEscalation** - Gates whether or not a user is allowed to set the
+security context of a container to `allowPrivilegeEscalation=true`. This
+defaults to allowed.
+
+**DefaultAllowPrivilegeEscalation** - Sets the default for the
+`AllowPrivilegeEscalation` option. The default behavior is to allow privilege
```

Comment:
Better clarify the `AllowPrivilegeEscalation` here. Does it mean the field of PodSecurityPolicy or the field 'pod.spec.container[i].securityContext.allowPrivilegeEscalation' ?

Reply:
It's explained above the fields, since it applies to both fields. Would you prefer I give the full path instead of `container option`?

Reply:
I tried to clarify it. Better?

> the AllowPrivilegeEscalation field of the PodSecurityPolicy is unspecified

If it's set to `true`, this field can still be used to default it to false.